packages/app/src/cli/services/build/extension.ts

```diff
@@ -54,6 +56,47 @@ export async function buildThemeExtensions(options: ThemeExtensionBuildOptions):
   })
 }
 
+interface BuildUIExtensionsOptions {
+  app: AppInterface
+}
+
+export function buildUIExtensions(options: BuildUIExtensionsOptions): output.OutputProcess[] {
+  if (options.app.extensions.ui.length === 0) {
+    return []
+  }
+  if (environment.utilities.isTruthy(process.env.SHOPIFY_CLI_UI_EXTENSIONS_USE_NODE)) {
```

Comment:
should we have this kind of checks abstracted?
We have `local.ts` with a bunch of them but i guess only for cli-kit, maybe makes sense to make that file public, or have a similar one for `app`.
What do you think?

Reply:
Note that this is very temporary. It'll go away once we have everything migrated and well tested.